### PR TITLE
kola/validate-symlinks: skip `/usr/lib/firmware` symlinks

### PR DESCRIPTION
**Pull request by marmijo**
A recent rawhide package of linux-firmware caused the validate-symlinks kola tests to fail. Let's modify the broken-symlinks list to skip all of `/usr/lib/firmware` so we can unblock FCOS rawhide builds.

See: https://github.com/coreos/fedora-coreos-tracker/issues/1479

### CODE REVIEW

File: tests/kola/files/validate-symlinks [hunk 1]
```diff
@@ -18,7 +18,7 @@ list_broken_symlinks_skip=(
     '/etc/swid/swidtags.d/fedoraproject.org'
     '/etc/xdg/systemd/user'
     '/usr/lib/.build-id/'
-    '/usr/lib/firmware/qcom/LENOVO/21BX.xz'
+    '/usr/lib/firmware/'
```
Comment (gursewak1997) [inline tests/kola/files/validate-symlinks]: I just hope we keep `'/usr/lib/firmware/qcom/LENOVO/21BX.xz'` after we revert this change( hoping this gets fixed soon)

**Review by gursewak1997 — DISMISSED** [review]
lgtm

File: tests/kola/files/validate-symlinks (hunk 1 above)
Comment (marmijo) [reply tests/kola/files/validate-symlinks]: I agree. I wasnt sure about this one. I could just add `/usr/lib/firmware` as it's own line so there is no loss of info here.

Comment (gursewak1997) [reply tests/kola/files/validate-symlinks]: Sure, let's do that

**Comment by marmijo** [comment]
@gursewak1997 I went ahead and added it back to the list so we don't lose any information. cc @travier

**Review by gursewak1997 — DISMISSED** [review]

**Comment by travier** [comment]
LGTM as is. I think we can drop all firmware symlink tracking so dropping the Lenovo one would be fine.

**Review by travier — DISMISSED** [review]

**Comment by dustymabe** [comment]
I think what we've decided here is that the files under `/usr/lib/firmware/` are regularly broken and it's not an issue and we'll ignore those broken symlinks in the future.

If that's the case let's maybe add a comment indicating that and also maybe make the commit message to state it's a regular occurrence and we've decided we can ignore them.

File: tests/kola/files/validate-symlinks (hunk 1 above)
Comment (gursewak1997) [reply tests/kola/files/validate-symlinks]: Looks like you can go back to your initial commit and add a comment:)

**Comment by marmijo** [comment]
I've updated the commit and message to reflect that we're opting to skip validating sym links in the `/usr/lib/firmware` location altogether.

**Review by gursewak1997 — APPROVED** [review]

**Review by travier — APPROVED** [review]
